core/utils/utils.go

```diff
@@ -41,10 +41,14 @@ const (
 	EVMWordHexLen = EVMWordByteLen * 2
 )
 
-// ZeroAddress is an empty address, otherwise in Ethereum as
+// ZeroAddress is an address of all zeroes, otherwise in Ethereum as
 // 0x0000000000000000000000000000000000000000
 var ZeroAddress = common.Address{}
 
+// ZeroHash is a hash of all zeroes, otherwise in Ethereum as
+// 0x0000000000000000000000000000000000000000000000000000000000000000
+var EmptyHash = common.Hash{}
```

Comment:
EmptyHash has ZeroHash comment, would it be better to unify the names used here? Both Zero* or both Empty*?